lib/transformer.dart

```diff
@@ -98,6 +98,19 @@ class WebSkinDartTransformer extends Transformer implements LazyTransformer {
     TransformedSourceFile transformedFile = new TransformedSourceFile(sourceFile);
     TransformLogger logger = new JetBrainsFriendlyLogger(transform.logger);
 
+    var partPattern = new RegExp(r'''part\s+['"](.+.over_react.g.dart)['"];''');
+    var partFilename = partPattern.firstMatch(sourceFile.getText(0));
+
+    // For Dart 1 compatibility an empty generated part file will be created when a file contains
+    // the part directive pointing to the generated file the new builder requires.
+    if (partFilename != null) {
+        var sourceFileDirectory = p.dirname(sourceFile.url.toFilePath());
+        var partFilePath = p.join(sourceFileDirectory, partFilename.group(1));
+        var asset = new Asset.fromString(new AssetId(transform.primaryInput.id.package, partFilePath), '');
```

Comment:
Shouldn't this contain a `part of` directive?

Or, does it not matter since dart2js doesn't care? If so, fine by me 🤷‍♂️

Reply:
Yeah it should, good catch.

```dart
var contents = "part of '${p.basename(sourceFile.url.toFilePath())}';";
var asset = new Asset.fromString(new AssetId(transform.primaryInput.id.package, partFilePath), contents);
```

@sebastianmalysa-wf seb we'll want to do this ^